guides/release/tutorial/13.md

```diff
@@ -0,0 +1,93 @@
+Next, let's add the image for the rental property. We will use the component generator for this again.
+
+```
+ember generate component rental/image
+```
+
+This time, we had a slash in the component's name. This resulted in the component being created at `app/components/rental/image.hbs` instead of at the root of `app/components`. This component can be invoked as `<Rental::Image>`.
+
+Components like these are known as *[namespaced](TODO: link to namespaced)* components. They allow us to organize our components by folders according to their purpose. This is completely optional – namespaced components are not special in any way.
+
+Let's edit the component's template:
+
+```handlebars {data-filename="app/components/rental/image.hbs"}
+<div class="image">
+  <img ...attributes>
+</div>
+```
+
+Instead of hard-coding specific `src` and `alt` attributes on the `<img>` tag, we opted for the `...attributes` keyword instead, which is also sometimes referred to as *["splattributes"](TODO: link to splattributes)* syntax. This allows arbitrary HTML attributes to be passed in when invoking this component:
+
+```handlebars {data-filename="app/components/rental.hbs" data-diff="+2,+3,+4,+5,+6"}
+<article class="rental">
+  <Rental::Image
+    src="https://upload.wikimedia.org/wikipedia/commons/c/cb/Crane_estate_(5).jpg"
+    alt="A picture of Grand Old Mansion"
+  />
+
+  <div class="details">
+    <!-- ...snip... -->
+  </div>
+</article>
+```
+
+We specified a `src` and an `alt` HTML attribute here, which will be passed along to the component and attached to the element where `...attributes` is specified. You can think of this as being similar to `{{yield}}`, but for HTML attributes specifically, instead of for any content. In fact, we used this feature earlier when we passed a `class` attribute to `<LinkTo>`.
```

Comment:
```suggestion
We specified a `src` and an `alt` HTML attribute here, which will be passed along to the component and attached to the element where `...attributes` is invoked. You can think of this as being similar to `{{yield}}`, but for HTML attributes rather than displayed content. In fact, we used this feature earlier when we passed a `class` attribute to `<LinkTo>`.
```